source/sass/blocks/main-index/price-table.scss

```
@@ -0,0 +1,53 @@
+.price-table__category {
+  padding: 25px 67px;
+
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 30px;
+
+  color: $font-color-secondary;
+  background-color: $font-color-primary;
+  text-transform: uppercase;
+
+
+  p {
+      padding: 0;
+      margin: 15px 0 0 0;
+
+      font-weight: 300;
+      font-size: 24px;
+      line-height: 30px;
+    }
```

Что-то с отступами тут...
```suggestion
  p {
    padding: 0;
    margin: 15px 0 0 0;

    font-weight: 300;
    font-size: 24px;
    line-height: 30px;
  }
```